src/utils/fmt.zig

```diff
@@ -188,19 +188,19 @@ pub fn newlinesToSpaces(comptime str: []const u8) [str.len]u8 {
 /// Tries to format the real path resolved from `dir` and `pathname`.
 /// Should it encounter an error when doing so, `"(error.Name)/pathname"`
 /// is printed instead.
-pub inline fn tryRealPath(dir: std.fs.Dir, pathname: []const u8) TryRealPathFmt {
-    return .{
+pub inline fn tryRealPath(dir: std.fs.Dir, pathname: []const u8) std.fmt.Formatter(TryRealPathCtx.realPathFormat) {
```

Comment:
There's no point in using `std.fmt.Formatter` (I think it should be removed altogether). Any reason you did this?

Reply:
8948a0cb3b14df8d6d5d65356aefa394a1c27ee8